README.md

```diff
@@ -153,15 +153,15 @@ interface DelegatedInkTrailPresenter : InkPresenter {
 
 This proposal provides infrastructure for authors to request a generic `InkPresenter` interface. A delegated ink trail `InkPresenter` can be requested and provided by User Agents that support it. `requestPresenter` is intentionally made to be extensible via a parameter so that in the future, it can easily be extended to other types of ink presentation. For example, this could include fully delegated ink presentation, or a presenter than can handle complex tips via WebGL shaders.
 
-The `DelegatedInkTrailPresenter` `setLastRenderedPoint` method is main addition of this proposal. Authors should use this method to indicate to the User Agent which PointerEvent was used as the last rendered point for the current frame. The PointerEvent passed to `setLastRenderedPoint` must be a trusted event and should be the last point that was used to by the application to render ink to the view. `setLastRenderedPoint` also accepts all relevant properties of rendering the ink stroke so that the User Agent can closely match the ink rendered by the application. The `diameter` passed as part of `InkTrailStyle` is to indicate the width of the ink trail drawn by the User Agent.
+The `DelegatedInkTrailPresenter` `updateInkTrailStartPoint` method is main addition of this proposal. Authors should use this method to indicate to the User Agent which PointerEvent was used as the last rendered point for the current frame. The PointerEvent passed to `updateInkTrailStartPoint` must be a trusted event and should be the last point that was used to by the application to render ink to the view. `updateInkTrailStartPoint` also accepts all relevant properties of rendering the ink stroke so that the User Agent can closely match the ink rendered by the application. The `diameter` passed as part of `InkTrailStyle` is to indicate the width of the ink trail drawn by the User Agent.
```

Comment:
lets add one more sentence after the first one that mentions something like

"This PointerEvent (and related style information) will be used by the User Agent/Operating system as the starting point for which the delegated trail will be rendered for the subsequent frame that is produced."

Reply:
Or maybe this can be merged with the `updateInkTrailStartPoint` also accepts... sentence somehow

Reply:
Added, and moved the one awkward sentence about diameter into its own section about `InkTrailStyle` and how that is extensible for more properties in the future. Let me know what you think.